tests/test_list

```diff
@@ -9,3 +9,4 @@ test_sysbench.sh
 test_uperf.sh
 test_byowl.sh
 test_hammerdb.sh
+test_vdbench.sh
```

Comment:
this should be test_vdbenchd.sh as thats the file name? CI seems to be failing on this.

Reply:
This is the file, i am just looking in the log to see way it fail.

Reply:
you're missing the `d` in the name, that's why its failing. the line should be `test_vdbenchd.sh` instead of `test_vdbench.sh`, because your test script is named test_vdbenchd.sh

Reply:
yes, i just fixed it.
forgot the 'd' in the name